sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -104,7 +104,11 @@ def __init__(
 
     @property
     def headers(self) -> Dict[str, str]:
-        """The current headers collection."""
+        """The current headers collection.
+
+        :rtype: dict[str, str]
+        :return: The current headers collection.
```

Comment:
As I was updating identity, I ran into several `@property`-decorated methods that needed a `:return:` docstring, but got the feeling that it was superfluous, as it's generally repeating what is being said in the first line of the docstring.  On the docs site, these properties are listed as attributes ([example](https://learn.microsoft.com/en-us/python/api/azure-eventhub/azure.eventhub.eventdata?view=azure-python#attributes)), so these would seemingly add additional clutter without really adding value. 

Do you think it's worthwhile to adjust the pylint rule to not require a `:return:` on `@property` methods?

Reply:
Yes, I can edit the rule for this to avoid the repetition! Thanks for pointing that out